doc/tutorials/Chapter_1/01-Setup_Installation.md

```diff
@@ -0,0 +1,302 @@
+# ROHD Setup & Installation
+
+There are two options for developing with ROHD, either through running it on a GitHub Codespace or on your local machine.
```

Comment:
This is a long page, and at the top you state there are "two options", but there's three even though two of them are on the local machine.  Some sort of list or even a table of contents at the top of this page so that people can jump to the section they are interested in would be helpful.

Reply:
The table of content can be found at the root directory of the tutorials. Do you think we still want to include it on top of the page?